### DISCUSSION TITLE
Search icon in Auto-complete field

### DISCUSSION THREAD

**Discussion by KateMacdonald** [WestpacGEL/GEL-next#164] [Ideas]
Can we please add the same Icon functionality to the **Auto-complete input field** as we have in the new **Input field** component. Primarily to have a Search icon visible on the left.

![image](https://github.com/WestpacGEL/GEL-next/assets/6667337/fc6234a5-9e4c-414c-8145-09b47cfbf755)

**Comment by samithaf** [comment]
Ticket https://github.com/WestpacGEL/GEL-next/issues/564
